docs/guides/implementation.md

```diff
@@ -0,0 +1,65 @@
+# Implementation Guide
+
+This guide provides a roadmap of a typical implementation with Worklytics-provided support.
+
+## 1 Kick-off/Scoping meeting
+
+*30-60 min video call to get overview of process, responsibilities*
+
+Attendees:
+  - Product Stakeholder(s)
+  - Data Source Administrator(s), if identified
+  - IT Admin(s), if identified
+
+Agenda:
+ - determine data sources, and who can authorize access to each
+ - determine host platform (GCP or AWS)
+ - identify who has the permissions to manage infra, will be able to run Terraform, and how
+   they'll run it (where, authenticated how)
+ - scope desired data interval, approximate headcount, etc.
+ - identify any potential integration issues or infrastructure constraints
+
+## 2 Initial Walk through
+
+*1-2 hr video call, to walk-through customization and initial terraform runs via screenshare*
+
+Attendees:
+  - IT Admin(s) who will be running Terraform
+  - Worklytics technical contact
+
+Prior to this call, please follow the initial steps in the `Getting Started` section for your host
+platform and ensure you have all Prereqs
+
+Goals:
+  1. get example customized and a terraform plan working.
+  2. run `terraform apply`. Obtain the `TODO 1` files you can send to your data source
+     administrators to complete, as needed.
+
+Tips:
+  - Works best if we screenshare
+
+## 3 Testing / Validation
+
+*can be completed without call; but Worklytics can assist if desired*
+
+  - follow `TODO 2` files / use test.sh shell scripts produced by `terraform apply`
+  - validate that authentication/authorization is correct for all connections, and that you're
+    satisfied with proxy behavior
+
+## 4 Authorize Worklytics to Access Sanitized Data
+
+*can be completed without call; but Worklytics can assist if desired*
+
+Authorize Worklytics to invoke API connectors / access sanitized bulk data:
+  - obtain service account ID of your tenant from Worklytics; configure it in you terraform.tfvars file
+  - run `terraform apply` again to update IAM policy to reflect the change
+
+## 5 Connect Sanitized Data Sources to Worklytics
+
+*can be completed without call; but Worklytics can assist if desired*
+
+  - follow `TODO 3` files (or terraform output values) generated by the `terraform apply` command
+  - if you do  not have access to Worklytics or do not have `Data Connection Admin` role, send
```

Comment:
```suggestion
  - if you do not have access to [Worklytics](https://app.worklytics.co),  or you do, but do not have `Data Connection Admin` role, send
```
Also, maybe worthy to add a link entry about configuring SSO if no access available? (https://docs.worklytics.co/knowledge-base/access-control/identity-providers)